packages/components/codemods/Well/transformWell.spec.ts

```diff
@@ -0,0 +1,186 @@
+import path from "path"
+import ts from "typescript"
+import { transformWellSource, updateFileContents } from "./transformWell"
+
+// This function is used to parse a stringified JSX element into an AST
+function parseJsx(jsx: string): ts.SourceFile {
+  return ts.createSourceFile(
+    "tempFile.tsx",
+    jsx,
+    ts.ScriptTarget.Latest,
+    true,
+    ts.ScriptKind.TSX
+  )
+}
+
+// This function is used to print the AST as a string and is only used for testing
+function printAst(ast: ts.SourceFile): string {
+  const printer = ts.createPrinter()
+  return printer.printFile(ast)
+}
```

Comment:
Maybe these can be moved to a shared space eg. `codemods/__tests__/utils(or individual files)`? :)